lib/portlayer/network/scope.go

```diff
@@ -161,6 +161,7 @@ func (s *Scope) reserveEndpointIP(e *Endpoint) error {
 				e.ip = eip
 				return nil
 			}
+			err = fmt.Errorf("No available IP address in current network %s", s.name)
```

Comment:
What is the reason to throw a new error here?

Reply:
More friendly :)